sidebars.js

```diff
@@ -139,19 +139,39 @@ const sidebars = {
         "subnets/troubleshoot-deployments",
       ],
     },
+    {
+    type: "category",
+    label: "Custom Subnet",
+    collapsed: true,
+    items: [  
+      "subnets/customize-a-subnet",
+      "subnets/cross-subnet-communication"
+    ]    
+    },
+    {
+      type: "category",
+      label: "Custom Subnet-EVM chain",
+      collapsed: true,
+      items: [
+
+        "subnets/customize-a-subnet-evm-chain",
+        "subnets/subnet-evm-genesis-file",
+        "subnets/subnet-evm-precompiles",
+        "subnets/subnet-evm-chain-configs",
+        "subnets/subnet-evm-network-upgrades",
+        "subnets/hello-world-precompile-tutorial"
+      ],
+    },
     {
       type: "category",
-      label: "Customize",
+      label: "Custom VM Chain",
```

Comment:
I wound suggest this should be "Build a VM Chain", not `customize` as all the docs under this tab is `how to build ....`